tutorials/market/Market_Intelligence_Part2.ipynb

```diff
@@ -55,18 +55,21 @@
     "#     authenticator=ibm_cloud_sdk_core.authenticators.IAMAuthenticator(api_key)\n",
```

Comment:
<p>I think the transitive closure link didn't render correctly.</p><p> </p><p>Also, it would be helpful to say how exactly you get the job title after step 3. I think it's just putting the set of nodes together, but worth mentioning I think.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/188/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>